swaglyrics/__main__.py

```diff
@@ -30,9 +30,15 @@ def main():
 	parser.add_argument('-t', '--tab', action='store_true', help='Display lyrics in a browser tab.')
 	parser.add_argument('-c', '--cli', action='store_true', help='Display lyrics in the command-line.')
 
+	parser.add_argument('--song')
+	parser.add_argument('--artist')
+
 	args = parser.parse_args()
+	print(args)
```

Comment:
I'm assuming this was a debug statement?

Reply:
Yeah, right